src/utils/multisig.ts

```diff
@@ -170,37 +170,96 @@ export function isDevKeyChangeDetailed(oldConfig: any, newConfig: any): boolean
   return isKeyChangeDetailed(oldConfig, newConfig, 'devKeys')
 }
 
+/**
+ * Normalizes an Ethereum address to a consistent format for comparison
+ * 
+ * @param address The Ethereum address to normalize
+ * @returns The normalized Ethereum address
+ */
+export function normalizeEthAddress(address: string): string {
+  if (!address) {
+    return '';
+  }
+
+  let normalized = address.trim();
+  
+  // Add 0x prefix if missing
+  if (!normalized.toLowerCase().startsWith('0x')) {
+    normalized = '0x' + normalized;
+  } else if (normalized.startsWith('0X')) {
+    // Ensure consistent 0x prefix (not 0X)
+    normalized = '0x' + normalized.substring(2);
+  }
```

Comment:
nitpick: this condition isn't needed as we are already lowercasing at line 196 below